src/languages/swift.js

```diff
@@ -457,6 +457,37 @@ export default function(hljs) {
     end: /}/
   };
 
+  const TYPE_DECLARATION = {
+    match: [
+      /(struct|protocol|class|extension|enum|actor)/,
+      /\s+/,
+      /[A-Za-z$_][\u00C0-\u02B80-9A-Za-z$_]*/,
```

Comment:
What is this?  I'm wondering if perhaps it should be a named constant that we pull in from `Swift` like we do with other major patterns?

Reply:
This was the existing regex used when a type name is declared. However, I've had another look at [the grammar](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/summaryofthegrammar/#Declarations) and it appears that a type declaration is actually the same as `Swift.identifier` in this case. I'll update this.